SU-HKKU/cardinal_service_profile
# CSD-238 Built minimal card paragraph type

# READY FOR REVIEW

# Summary
- Minimal card paragraph type

# Need Review By (Date)
- 7/8

# Urgency
- medium

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-238`
1. `blt drupal:sync --site=cardinalservice`
1. create a basic page
1. add a couple minimal cards
1. verify they look like the [mockups](https://www.figma.com/file/CpblbZldiGM6oYawa9aB4Q/Look-%26-Feel?node-id=1%3A2128) (home page below the top banner.)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
